terraform/20-app/ecs.service.feature-flags.tf

```diff
@@ -0,0 +1,120 @@
+module "ecs_service_feature_flags" {
+  source  = "terraform-aws-modules/ecs/aws//modules/service"
+  version = "5.2.0"
+
+  name                   = "${local.prefix}-feature-flags"
+  cluster_arn            = module.ecs.cluster_arn
+  enable_execute_command = true
+
+  create_iam_role = true
+
+  cpu        = 256
+  memory     = 512
+  subnet_ids = module.vpc.private_subnets
+
+  enable_autoscaling       = false
+  desired_count            = 1
+  autoscaling_min_capacity = 1
+  autoscaling_max_capacity = 1
+
+  container_definitions = {
+    api = {
+      cloudwatch_log_group_retention_in_days = local.default_log_retention_in_days
+      cpu                                    = 256
+      memory                                 = 512
+      essential                              = true
+      readonly_root_filesystem               = false
+      image                                  = "unleashorg/unleash-server:5.10.1"
```

Comment:
Not ideal going directly to Docker hub.
Once unleash mirror this image into their own official repo in the public AWS ECR gallery, then we can just pull from there instead. If this takes too long and becomes too much of a risk, we can set up a pull through cache in the tools account and nab the images from there instead